CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ Changelog
 
 ### Other Changes
 
-* upgrade to OSHDB version 0.7.0 – note that oshdb database files from the previous version are not compatible with this version anymore, you have to either [redownload](https://downloads.ohsome.org/OSHDB/v0.7/) or [recreate](https://github.com/GIScience/oshdb/blob/0.7/oshdb-etl/README.md) them from scratch ([#222])
+* upgrade to OSHDB version 0.7 – note that oshdb database files from the previous version are not compatible with this version anymore, you have to either [redownload](https://downloads.ohsome.org/OSHDB/v0.7/) or [recreate](https://github.com/GIScience/oshdb/blob/0.7/oshdb-etl/README.md) them from scratch ([#222])
```

Comment:
Don't we want to state the explicit OSHDB version here? I would also think linking a tag instead of a release branch would be better, wouldn't it?

Reply:
I thought that if we write "upgrade to OSHDB version 0.7.1", it could be read as that it implies that we previously had used 0.7.0. Also, some people might think that the need to redownload OSHDB database files is specific to the particular version 0.7.1 (and would not be required for 0.7.0 for example).

What about something like the following?
```suggestion
* upgrade to OSHDB version 0.7.1 – note that OSHDB database files from the previous version are not compatible with this the OSHDB version 0.7 anymore, you have to either [redownload](https://downloads.ohsome.org/OSHDB/v0.7/) or [recreate](https://github.com/GIScience/oshdb/blob/0.7/oshdb-etl/README.md) them from scratch ([#222])
```

Reply:
Sounds good, but you have a `this` to much in your suggestion.